src/util/jwt_utils.js

```diff
@@ -27,11 +27,35 @@ function make_internal_auth_token(object = {}, jwt_options = {}) {
     return make_auth_token(object, jwt_options);
 }
 
+/**
+ * authorize jwt token. Validate the jwt token inside passed validate_jwt method
+ * @param {string} token
+ */
 function authorize_jwt_token(token) {
+    if (token && token.includes('Bearer ')) {
+        token = token.split(' ')[1];
+    }
```

Comment:
this better move to the calling function, so that a token here is just a jwt token and not the full header.